### DISCUSSION TITLE
`gtk-libadwaita` vs `gtk-adwaita`

### DISCUSSION THREAD

**Discussion by jcollie** [ghostty-org/ghostty#2574] [Feature Requests, Ideas] [answered]
In `build.zig` the `gtk-libadwaita` build option is defined:
https://github.com/ghostty-org/ghostty/blob/d47df211b03894427592851b0be3b528ff8221ff/build.zig#L94-L98

In `src/config/Config.zig` a config option named `gtk-adwaita` is defined:
https://github.com/ghostty-org/ghostty/blob/d47df211b03894427592851b0be3b528ff8221ff/src/config/Config.zig#L1626-L1638

I find it slightly annoying that these are named differently, what do people think about renaming one or the other so that they are the same?

**Comment by mitchellh** [comment, marked as answer]
I'm good with this. I won't create an issue since this is so trivial and I have a feeling you're going to do it quickly. 😄 But go ahead and lets make that breaking change before the 1.0 release.
